src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/ChatWithCustomData/ChatWithCustomData.Web/Components/Pages/Chat/Chat.razor

```diff
@@ -23,9 +23,13 @@
         Do not answer questions about anything else.
         Use only simple markdown to format your responses.
 
-        Use the search tool to find relevant information. When you do this, end your
-        reply with citations in the special format, always formatted as XML:
-        <citation filename='string' page_number='number'>exact quote here</citation>.
+         Use the search tool to find relevant information. When you do this, end your
+        reply with citations in the special XML format:
+
+        <citation filename='string' page_number='number'>exact quote here</citation>
+
+         Always include the citation in your response if there are results.
```

Comment:
Nbd but something quirky about the indentation. Not sure if it's a tabs/spaces thing:

```suggestion
        Use the search tool to find relevant information. When you do this, end your
        reply with citations in the special XML format:

        <citation filename='string' page_number='number'>exact quote here</citation>

        Always include the citation in your response if there are results.
```